.storybook/public/mockServiceWorker.js

```diff
@@ -0,0 +1,338 @@
+/* eslint-disable */
```

Comment:
Er det noen fordeler med å forvalte dette data-fetching oppsettet vs å bare ha en random timeout i table-story? Kan hende jeg har misset contexten for hvorfor det er satt opp.

Reply:
Jeg tenker det er nice at eksempelet blir så likt som mulig et reelt case.